object.go

```diff
@@ -80,12 +80,13 @@ type ComponentTypeId struct {
 // ComponentType Information about a particular component type
 type ComponentType struct {
 	ComponentTypeId
-	Description string            // The description of the component type (Optional)
-	Href        string            // The relative path to link to the component type (Required)
-	Icon        ComponentTypeIcon // The icon associated with the component type (Required)
-	IsDefault   bool              // Whether or not the component type is the default (Required)
-	Name        string            // The name of the component type (Required)
-	Timestamps  Timestamps        // When the component type was created and updated (Required)
+	Description string                        // The description of the component type (Optional)
+	Href        string                        // The relative path to link to the component type (Required)
+	Icon        ComponentTypeIcon             // The icon associated with the component type (Required)
+	IsDefault   bool                          // Whether or not the component type is the default (Required)
+	Name        string                        // The name of the component type (Required)
+	Timestamps  Timestamps                    // When the component type was created and updated (Required)
+	Properties  *PropertyDefinitionConnection `graphql:"-"`
```

Comment:
This will get to get ported into codegen but this effectively tells the graphql client to ingore this field since we'll populate it with `GetProperties` when called